app/models/showcase.js

```diff
@@ -0,0 +1,10 @@
+import DS from 'ember-data';
```

Comment:
Should we do `import attr from 'ember-data/attr'` instead?

Reply:
@abhilashlr we should follow the default recommendation.

Reply:
Yea I think you're right @locks 👍 https://guides.emberjs.com/release/models/defining-models/#toc_defining-attributes